connection.go

```diff
@@ -80,12 +82,12 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 	}
 }
 
-// Connection is a handle to Tarantool.
+// Connection is a handle with a single connection to Tarantool instance.
 //
 // It is created and configured with Connect function, and could not be
 // reconfigured later.
 //
-// It is could be "Connected", "Disconnected", and "Closed".
+// It could be in "Connected", "Disconnected", and "Closed" states.
 //
 // When "Connected" it sends queries to Tarantool.
 //
```

Comment:
> Connection could become "Closed"...

is really hard to read, a list would be more preferable here.

Reply:
rewrote it